### PR TITLE
Use secure endpoint to fetch the license

### PR DESCRIPTION
**Pull request by dborovcanin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,18 +70,17 @@ The agent is configured using the environment variables presented in the
...
-| MF_AGENT_PORT               | Agent port                                                        |                                  |
-| MF_AGENT_LOAD_RETRY_SECONDS | Retry period in seconds to wait between two sync calls to service | 60                               |
-| MF_AGENT_LICENSE_ID         | Agent License ID                                                  |                                  |
-
-The Agent fetches the License using it's ID, and exposes HTTP API for validation. You can se the License validation flow on the diagram below:
+| Variable                    | Description                                                       | Default                                  |
+| --------------------------- | ----------------------------------------------------------------- | ---------------------------------------- |
+| MF_LICENSE_LOG_LEVEL        | Agent log level                                                   | "error"                                  |
+| MF_LICENSE_SERVICE_URL      | License service URL                                               | "http://localhost:8180/licenses/devices" |
```
Comment (manuio) [inline README.md]: Port should be `8111`, right? Maybe this could be simply `localhost:8111`?

**Review by manuio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dborovcanin) [reply README.md]: It can not be `localhost:8111` because it is an HTTP endpoint, but it should be `8111` port. Let me fix it.

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**